scrapers/TheNude.yml

```diff
@@ -0,0 +1,76 @@
+name: TheNude
+performerByName:
+  action: scrapeXPath
+  queryURL: https://www.thenude.com/index.php?page=search&action=searchModels&__form_name=navbar-search&m_aka=on&m_name={}
+  scraper: performerSearch
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.thenude.com
+    scraper: performerScraper
+xPathScrapers:
+  performerSearch:
+    performer:
+#      Name: //a[@class="model-name"]/text()
+      Name: //a[@class="model-name"]/../../a/@title

```

Comment:
another alternative for the names shown when searching
```yaml
      Name:
        selector: //figcaption/span
        replace:
          - regex: "^AKA:"
            with: ""
```
i don't know which is better though

Reply:
I can't decide as well between the three possibilities:
1. `//a[@class="model-name"]/text()`  gets only the name
2. `//a[@class="model-name"]/../../a/@title`  gives more details on the label / studio 
3. `//figcaption/span` gives a whole bunch of aliases as well.

So what would be the expected behavior for which workflow?

Reply:
If you don't exactly know what you are looking for, option 3 seems to be the best

Reply:
only the name is consistent with the rest of the performer scrapers, but doesn't help much
label and studio can be helpful to get the exact performer you want 
all aliases help when you don't know the studio,label

so all are useful
pick anyone you want and leave the others as a comment, then the user can select based on his usage patterns.